src/actions/psdTools/ClipAction.ts

```diff
@@ -5,14 +5,33 @@ import {clip, clipEvenOdd} from "../../values/flag";
 
 /**
  * @description  Defines the clipping path to use when trimming pixels.
- * @augments Action
+ * @memberOf Actions.PSDTools
+ * @extends {SDK.Action}
  */
 class ClipAction extends Action {
   private path: string | number;
   private isEvenOdd = false;
-  constructor(path: string|number) {
+  constructor() {
     super();
+
+  }
+
+  /**
+   * @description The name of the path to clip by
```

Comment:
is this missing something?

Reply:
I'm not sure what you mean, but I added return values in the JSDoc strings.